scalameter-core/src/main/scala/org/scalameter/PicklerBasedKey.scala

```diff
@@ -0,0 +1,15 @@
+package org.scalameter
+
+import org.scalameter.picklers.Pickler
+
+
+/* A mixin for keys whose values require pickler based serialization. */
+trait PicklerBasedKey[T] {
```

Comment:
Oh, my bad. Yes, it should be. 
I suppose that it wasn't causing any error because Parameter is a `case class` and `Key` was extending Serializable already.